Description: Fix some compiler warnings
 This fixes a FTBFS with GCC 6 where the warnings are now errors.
Bug-Debian: https://bugs.debian.org/812167
Origin: upstream, https://bitbucket.org/giszmo/glob2/commits/a9a16d8631db0176267b4f4aefd8e93767f8d247
Author: Leo Wandersleb <Leo.Wandersleb@gmx.de>
Last-Update: 2011-11-03
--- a/src/ChooseMapScreen.cpp
+++ b/src/ChooseMapScreen.cpp
@@ -31,7 +31,7 @@
 
 #include "Game.h"
 
-ChooseMapScreen::ChooseMapScreen(const char *directory, const char *extension, bool recurse, const char* alternateDirectory, const char* alternateExtension, const char* alternateRecurse)
+ChooseMapScreen::ChooseMapScreen(const char *directory, const char *extension, bool recurse, const char* alternateDirectory, const char* alternateExtension, const bool alternateRecurse)
 {
 	ok = new TextButton(440, 360, 180, 40, ALIGN_SCREEN_CENTERED, ALIGN_SCREEN_CENTERED, "menu", Toolkit::getStringTable()->getString("[ok]"), OK, 13);
 	addWidget(ok);
--- a/src/ChooseMapScreen.h
+++ b/src/ChooseMapScreen.h
@@ -43,7 +43,7 @@
 	/// Constructor. Directory is the source of the listed files.
 	/// extension is the file extension to show. If recurse is true,
 	/// subdirectoried are shown and can be opened.
-	ChooseMapScreen(const char *directory, const char *extension, bool recurse, const char* alternateDirectory=NULL, const char* alternateExtension=NULL, const char* alternateRecurse=NULL);
+	ChooseMapScreen(const char *directory, const char *extension, bool recurse, const char* alternateDirectory=NULL, const char* alternateExtension=NULL, const bool alternateRecurse=NULL);
 	//! Destructor
 	virtual ~ChooseMapScreen();
 	virtual void onAction(Widget *source, Action action, int par1, int par2);
